### ISSUE TITLE
Server has stopped responding 

### ISSUE THREAD

**Issue by jackcarrdev** [PaperMC/Paper#2007]
### Stacktrace:
```
[23:54:53 ERROR]: ------------------------------
[23:54:53 ERROR]: The server has stopped responding! This is (probably) not a Paper bug.
[23:54:53 ERROR]: If you see a plugin in the Server thread dump below, then please report it to that author
[23:54:53 ERROR]: *Especially* if it looks like HTTP or MySQL operations are occurring
[23:54:53 ERROR]: If you see a world save or edit, then it means you did far more than your server can handle at once
[23:54:53 ERROR]: If this is the case, consider increasing timeout-time in spigot.yml but note that this will replace the crash with LARGE lag spikes
[23:54:53 ERROR]: If you are unsure or still think this is a Paper bug, please report this to https://github.com/PaperMC/Paper/issues
[23:54:53 ERROR]: Be sure to include ALL relevant console errors and Minecraft crash reports
[23:54:53 ERROR]: Paper version: git-Paper-624 (MC: 1.13.2)
[23:54:53 ERROR]: ------------------------------
[23:54:53 ERROR]: Server thread dump (Look for plugins here before reporting to Paper!):
[23:54:53 ERROR]: ------------------------------
[23:54:53 ERROR]: Current Thread: Server thread
[23:54:53 ERROR]: PID: 20 | Suspended: false | Native: false | State: RUNNABLE
[23:54:53 ERROR]: Stack:
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.ItemStack.a(ItemStack.java:334)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.PlayerInventory.a(PlayerInventory.java:443)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.EntityHuman.b(EntityHuman.java:653)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.Block.getDamage(Block.java:412)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.IBlockData.getDamage(IBlockData.java:143)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:172)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1271)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.PacketPlayInBlockDig.a(SourceFile:40)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.PacketPlayInBlockDig.a(SourceFile:10)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:15)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.PlayerConnectionUtils$$Lambda$1924/1905237767.run(Unknown Source)
[23:54:53 ERROR]: java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[23:54:53 ERROR]: java.util.concurrent.FutureTask.run(FutureTask.java:266)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1018)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940)
[23:54:53 ERROR]: net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837)
[23:54:53 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:53 ERROR]: ------------------------------
[23:54:53 ERROR]: Entire Thread Dump:
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: pool-8-thread-1
[23:54:54 ERROR]: PID: 62 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.misc.Unsafe.park(Native Method)
[23:54:54 ERROR]: java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[23:54:54 ERROR]: java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[23:54:54 ERROR]: java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Netty Epoll Server IO #3
[23:54:54 ERROR]: PID: 58 | Suspended: false | Native: true | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: io.netty.channel.epoll.Native.epollWait0(Native Method)
[23:54:54 ERROR]: io.netty.channel.epoll.Native.epollWait(Native.java:114)
[23:54:54 ERROR]: io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:239)
[23:54:54 ERROR]: io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:256)
[23:54:54 ERROR]: io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Netty Epoll Server IO #2
[23:54:54 ERROR]: PID: 57 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[23:54:54 ERROR]: sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[23:54:54 ERROR]: sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[23:54:54 ERROR]: java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[23:54:54 ERROR]: java.lang.Class.newInstance(Class.java:442)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.EnumProtocol.a(SourceFile:346)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketDecoder.decode(SourceFile:34)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:310)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:284)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.LegacyPingHandler.channelRead(LegacyPingHandler.java:120)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1434)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:965)
[23:54:54 ERROR]: io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:808)
[23:54:54 ERROR]: io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:408)
[23:54:54 ERROR]: io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:308)
[23:54:54 ERROR]: io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Netty Epoll Server IO #1
[23:54:54 ERROR]: PID: 56 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketDataSerializer.d(PacketDataSerializer.java:192)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketPrepender.encode(SourceFile:25)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketPrepender.encode(SourceFile:8)
[23:54:54 ERROR]: io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:738)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:730)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:816)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:723)
[23:54:54 ERROR]: io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:113)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:738)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:801)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:814)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:794)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:831)
[23:54:54 ERROR]: io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1071)
[23:54:54 ERROR]: io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:300)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.NetworkManager.b(NetworkManager.java:201)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.NetworkManager.sendPacket(NetworkManager.java:172)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.NetworkManager.sendPacket(NetworkManager.java:166)
[23:54:54 ERROR]: com.destroystokyo.paper.network.StandardPaperServerListPingEventImpl.processRequest(StandardPaperServerListPingEventImpl.java:109)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketStatusListener.a(PacketStatusListener.java:131)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketStatusInStart.a(SourceFile:23)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketStatusInStart.a(SourceFile:8)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.NetworkManager.a(NetworkManager.java:156)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.NetworkManager.channelRead0(NetworkManager.java:145)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.NetworkManager.channelRead0(NetworkManager.java:31)
[23:54:54 ERROR]: io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:105)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:310)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:284)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:310)
[23:54:54 ERROR]: io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:284)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.LegacyPingHandler.channelRead(LegacyPingHandler.java:120)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
[23:54:54 ERROR]: io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1434)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
[23:54:54 ERROR]: io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
[23:54:54 ERROR]: io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:965)
[23:54:54 ERROR]: io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:808)
[23:54:54 ERROR]: io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:408)
[23:54:54 ERROR]: io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:308)
[23:54:54 ERROR]: io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Java2D Disposer
[23:54:54 ERROR]: PID: 54 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:144)
[23:54:54 ERROR]: java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:165)
[23:54:54 ERROR]: sun.java2d.Disposer.run(Disposer.java:148)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: File IO Thread
[23:54:54 ERROR]: PID: 53 | Suspended: false | Native: false | State: TIMED_WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Thread.sleep(Native Method)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.FileIOThread.c(FileIOThread.java:60)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.FileIOThread.run(FileIOThread.java:32)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Craft Async Scheduler Management Thread
[23:54:54 ERROR]: PID: 52 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.misc.Unsafe.park(Native Method)
[23:54:54 ERROR]: java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[23:54:54 ERROR]: java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[23:54:54 ERROR]: java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Query Listener #1
[23:54:54 ERROR]: PID: 51 | Suspended: false | Native: true | State: RUNNABLE
[23:54:54 ERROR]: Thread is waiting on monitor(s):
[23:54:54 ERROR]: Locked on:java.net.PlainDatagramSocketImpl.receive0(Native Method)
[23:54:54 ERROR]: Locked on:java.net.AbstractPlainDatagramSocketImpl.receive(AbstractPlainDatagramSocketImpl.java:143)
[23:54:54 ERROR]: Locked on:java.net.DatagramSocket.receive(DatagramSocket.java:812)
[23:54:54 ERROR]: Locked on:java.net.DatagramSocket.receive(DatagramSocket.java:812)
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.net.PlainDatagramSocketImpl.receive0(Native Method)
[23:54:54 ERROR]: java.net.AbstractPlainDatagramSocketImpl.receive(AbstractPlainDatagramSocketImpl.java:143)
[23:54:54 ERROR]: java.net.DatagramSocket.receive(DatagramSocket.java:812)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.RemoteStatusListener.run(RemoteStatusListener.java:335)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: pool-10-thread-2 Acceptor0 SelectChannelConnector@0.0.0.0:8123
[23:54:54 ERROR]: PID: 50 | Suspended: false | Native: true | State: RUNNABLE
[23:54:54 ERROR]: Thread is waiting on monitor(s):
[23:54:54 ERROR]: Locked on:sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:250)
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.nio.ch.ServerSocketChannelImpl.accept0(Native Method)
[23:54:54 ERROR]: sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:422)
[23:54:54 ERROR]: sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:250)
[23:54:54 ERROR]: org.dynmap.jetty.server.nio.SelectChannelConnector.accept(SelectChannelConnector.java:109)
[23:54:54 ERROR]: org.dynmap.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:938)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: pool-10-thread-1 Selector0
[23:54:54 ERROR]: PID: 49 | Suspended: false | Native: true | State: RUNNABLE
[23:54:54 ERROR]: Thread is waiting on monitor(s):
[23:54:54 ERROR]: Locked on:sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
[23:54:54 ERROR]: Locked on:sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
[23:54:54 ERROR]: Locked on:sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.nio.ch.EPollArrayWrapper.epollWait(Native Method)
[23:54:54 ERROR]: sun.nio.ch.EPollArrayWrapper.poll(EPollArrayWrapper.java:269)
[23:54:54 ERROR]: sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:93)
[23:54:54 ERROR]: sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
[23:54:54 ERROR]: sun.nio.ch.SelectorImpl.select(SelectorImpl.java:97)
[23:54:54 ERROR]: org.dynmap.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:569)
[23:54:54 ERROR]: org.dynmap.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: HashSessionScavenger-0
[23:54:54 ERROR]: PID: 48 | Suspended: false | Native: false | State: TIMED_WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.util.TimerThread.mainLoop(Timer.java:552)
[23:54:54 ERROR]: java.util.TimerThread.run(Timer.java:505)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Dynmap Render Thread
[23:54:54 ERROR]: PID: 44 | Suspended: false | Native: false | State: TIMED_WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.misc.Unsafe.park(Native Method)
[23:54:54 ERROR]: java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
[23:54:54 ERROR]: java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2078)
[23:54:54 ERROR]: java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1093)
[23:54:54 ERROR]: java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:809)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Dynmap Render Thread
[23:54:54 ERROR]: PID: 43 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.misc.Unsafe.park(Native Method)
[23:54:54 ERROR]: java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[23:54:54 ERROR]: java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[23:54:54 ERROR]: java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1088)
[23:54:54 ERROR]: java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:809)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Dynmap Render Thread
[23:54:54 ERROR]: PID: 42 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Thread is waiting on monitor(s):
[23:54:54 ERROR]: Locked on:java.util.zip.Inflater.inflate(Inflater.java:259)
[23:54:54 ERROR]: Locked on:java.io.BufferedInputStream.read(BufferedInputStream.java:345)
[23:54:54 ERROR]: Locked on:org.dynmap.utils.ImageIOManager.imageIODecode(ImageIOManager.java:248)
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.util.zip.Inflater.inflateBytes(Native Method)
[23:54:54 ERROR]: java.util.zip.Inflater.inflate(Inflater.java:259)
[23:54:54 ERROR]: java.util.zip.InflaterInputStream.read(InflaterInputStream.java:152)
[23:54:54 ERROR]: java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
[23:54:54 ERROR]: java.io.BufferedInputStream.read1(BufferedInputStream.java:286)
[23:54:54 ERROR]: java.io.BufferedInputStream.read(BufferedInputStream.java:345)
[23:54:54 ERROR]: java.io.DataInputStream.readFully(DataInputStream.java:195)
[23:54:54 ERROR]: com.sun.imageio.plugins.png.PNGImageReader.decodePass(PNGImageReader.java:1119)
[23:54:54 ERROR]: com.sun.imageio.plugins.png.PNGImageReader.decodeImage(PNGImageReader.java:1223)
[23:54:54 ERROR]: com.sun.imageio.plugins.png.PNGImageReader.readImage(PNGImageReader.java:1338)
[23:54:54 ERROR]: com.sun.imageio.plugins.png.PNGImageReader.read(PNGImageReader.java:1614)
[23:54:54 ERROR]: javax.imageio.ImageIO.read(ImageIO.java:1448)
[23:54:54 ERROR]: javax.imageio.ImageIO.read(ImageIO.java:1352)
[23:54:54 ERROR]: org.dynmap.utils.ImageIOManager.imageIODecode(ImageIOManager.java:248)
[23:54:54 ERROR]: org.dynmap.DynmapWorld.processZoomFile(DynmapWorld.java:160)
[23:54:54 ERROR]: org.dynmap.DynmapWorld.freshenZoomOutFiles(DynmapWorld.java:114)
[23:54:54 ERROR]: org.dynmap.MapManager$DoZoomOutProcessing.run(MapManager.java:835)
[23:54:54 ERROR]: org.dynmap.MapManager$DynmapScheduledThreadPoolExecutor$2.run(MapManager.java:209)
[23:54:54 ERROR]: java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[23:54:54 ERROR]: java.util.concurrent.FutureTask.run(FutureTask.java:266)
[23:54:54 ERROR]: java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
[23:54:54 ERROR]: java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Thread-14
[23:54:54 ERROR]: PID: 41 | Suspended: false | Native: false | State: TIMED_WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Thread.sleep(Native Method)
[23:54:54 ERROR]: org.dynmap.AsynchronousQueue.sleep(AsynchronousQueue.java:148)
[23:54:54 ERROR]: org.dynmap.AsynchronousQueue.running(AsynchronousQueue.java:138)
[23:54:54 ERROR]: org.dynmap.AsynchronousQueue.access$000(AsynchronousQueue.java:9)
[23:54:54 ERROR]: org.dynmap.AsynchronousQueue$1.run(AsynchronousQueue.java:83)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkGen-world_the_end
[23:54:54 ERROR]: PID: 39 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkGen-world_nether
[23:54:54 ERROR]: PID: 38 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkGen-world
[23:54:54 ERROR]: PID: 37 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkLoader-6
[23:54:54 ERROR]: PID: 36 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkLoader-5
[23:54:54 ERROR]: PID: 35 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkLoader-4
[23:54:54 ERROR]: PID: 34 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkLoader-3
[23:54:54 ERROR]: PID: 33 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkLoader-2
[23:54:54 ERROR]: PID: 32 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: PaperChunkLoader-1
[23:54:54 ERROR]: PID: 31 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor.processQueues(PriorityQueuedExecutor.java:213)
[23:54:54 ERROR]: com.destroystokyo.paper.util.PriorityQueuedExecutor$$Lambda$1715/1862208552.run(Unknown Source)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Netty Epoll Server IO #0
[23:54:54 ERROR]: PID: 30 | Suspended: false | Native: true | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: io.netty.channel.epoll.Native.epollWait0(Native Method)
[23:54:54 ERROR]: io.netty.channel.epoll.Native.epollWait(Native.java:114)
[23:54:54 ERROR]: io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:239)
[23:54:54 ERROR]: io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:256)
[23:54:54 ERROR]: io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:884)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: ObjectCleanerThread
[23:54:54 ERROR]: PID: 29 | Suspended: false | Native: false | State: TIMED_WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:144)
[23:54:54 ERROR]: io.netty.util.internal.ObjectCleaner$1.run(ObjectCleaner.java:54)
[23:54:54 ERROR]: io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Paper Watchdog Thread
[23:54:54 ERROR]: PID: 28 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.management.ThreadImpl.dumpThreads0(Native Method)
[23:54:54 ERROR]: sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:454)
[23:54:54 ERROR]: org.spigotmc.WatchdogThread.run(WatchdogThread.java:121)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Server console handler
[23:54:54 ERROR]: PID: 23 | Suspended: false | Native: true | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.io.FileInputStream.read0(Native Method)
[23:54:54 ERROR]: java.io.FileInputStream.read(FileInputStream.java:207)
[23:54:54 ERROR]: org.jline.terminal.impl.AbstractPty$PtyInputStream.read(AbstractPty.java:65)
[23:54:54 ERROR]: org.jline.utils.NonBlockingInputStream.read(NonBlockingInputStream.java:62)
[23:54:54 ERROR]: org.jline.utils.NonBlocking$NonBlockingInputStreamReader.read(NonBlocking.java:168)
[23:54:54 ERROR]: org.jline.utils.NonBlockingReader.read(NonBlockingReader.java:57)
[23:54:54 ERROR]: org.jline.keymap.BindingReader.readCharacter(BindingReader.java:133)
[23:54:54 ERROR]: org.jline.keymap.BindingReader.readBinding(BindingReader.java:110)
[23:54:54 ERROR]: org.jline.keymap.BindingReader.readBinding(BindingReader.java:61)
[23:54:54 ERROR]: org.jline.reader.impl.LineReaderImpl.readBinding(LineReaderImpl.java:786)
[23:54:54 ERROR]: org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:558)
[23:54:54 ERROR]: org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:404)
[23:54:54 ERROR]: net.minecrell.terminalconsole.SimpleTerminalConsole.readCommands(SimpleTerminalConsole.java:157)
[23:54:54 ERROR]: net.minecrell.terminalconsole.SimpleTerminalConsole.start(SimpleTerminalConsole.java:140)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.DedicatedServer$2.run(DedicatedServer.java:83)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: DestroyJavaVM
[23:54:54 ERROR]: PID: 22 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Server thread
[23:54:54 ERROR]: PID: 20 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PlayerConnection.isDisconnected(PlayerConnection.java:2635)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.EntityPlayer.isFrozen(EntityPlayer.java:1674)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1799)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketPlayInArmAnimation.a(SourceFile:32)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PacketPlayInArmAnimation.a(SourceFile:9)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:15)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.PlayerConnectionUtils$$Lambda$1924/1905237767.run(Unknown Source)
[23:54:54 ERROR]: java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[23:54:54 ERROR]: java.util.concurrent.FutureTask.run(FutureTask.java:266)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1018)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Server Infinisleeper
[23:54:54 ERROR]: PID: 21 | Suspended: false | Native: false | State: TIMED_WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Thread.sleep(Native Method)
[23:54:54 ERROR]: net.minecraft.server.v1_13_R2.DedicatedServer$1.run(DedicatedServer.java:66)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Snooper Timer
[23:54:54 ERROR]: PID: 18 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: java.util.TimerThread.mainLoop(Timer.java:526)
[23:54:54 ERROR]: java.util.TimerThread.run(Timer.java:505)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Paper Async Task Handler Thread - 0
[23:54:54 ERROR]: PID: 17 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: sun.misc.Unsafe.park(Native Method)
[23:54:54 ERROR]: java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
[23:54:54 ERROR]: java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
[23:54:54 ERROR]: java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Log4j2-TF-1-AsyncLogger[AsyncContext@1b6d3586]-1
[23:54:54 ERROR]: PID: 10 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Thread is waiting on monitor(s):
[23:54:54 ERROR]: Locked on:org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager.writeToDestination(RollingRandomAccessFileManager.java:109)
[23:54:54 ERROR]: Locked on:org.apache.logging.log4j.core.appender.OutputStreamManager.flushBuffer(OutputStreamManager.java:288)
[23:54:54 ERROR]: Locked on:org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager.flush(RollingRandomAccessFileManager.java:128)
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.io.RandomAccessFile.writeBytes(Native Method)
[23:54:54 ERROR]: java.io.RandomAccessFile.write(RandomAccessFile.java:525)
[23:54:54 ERROR]: org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager.writeToDestination(RollingRandomAccessFileManager.java:109)
[23:54:54 ERROR]: org.apache.logging.log4j.core.appender.OutputStreamManager.flushBuffer(OutputStreamManager.java:288)
[23:54:54 ERROR]: org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager.flush(RollingRandomAccessFileManager.java:128)
[23:54:54 ERROR]: org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.directEncodeEvent(AbstractOutputStreamAppender.java:179)
[23:54:54 ERROR]: org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.tryAppend(AbstractOutputStreamAppender.java:170)
[23:54:54 ERROR]: org.apache.logging.log4j.core.appender.AbstractOutputStreamAppender.append(AbstractOutputStreamAppender.java:161)
[23:54:54 ERROR]: org.apache.logging.log4j.core.appender.RollingRandomAccessFileAppender.append(RollingRandomAccessFileAppender.java:218)
[23:54:54 ERROR]: org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:156)
[23:54:54 ERROR]: org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:129)
[23:54:54 ERROR]: org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:120)
[23:54:54 ERROR]: org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:84)
[23:54:54 ERROR]: org.apache.logging.log4j.core.config.LoggerConfig.callAppenders(LoggerConfig.java:448)
[23:54:54 ERROR]: org.apache.logging.log4j.core.config.LoggerConfig.processLogEvent(LoggerConfig.java:433)
[23:54:54 ERROR]: org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:417)
[23:54:54 ERROR]: org.apache.logging.log4j.core.config.AwaitCompletionReliabilityStrategy.log(AwaitCompletionReliabilityStrategy.java:79)
[23:54:54 ERROR]: org.apache.logging.log4j.core.async.AsyncLogger.actualAsyncLog(AsyncLogger.java:337)
[23:54:54 ERROR]: org.apache.logging.log4j.core.async.RingBufferLogEvent.execute(RingBufferLogEvent.java:161)
[23:54:54 ERROR]: org.apache.logging.log4j.core.async.RingBufferLogEventHandler.onEvent(RingBufferLogEventHandler.java:45)
[23:54:54 ERROR]: org.apache.logging.log4j.core.async.RingBufferLogEventHandler.onEvent(RingBufferLogEventHandler.java:29)
[23:54:54 ERROR]: com.lmax.disruptor.BatchEventProcessor.processEvents(BatchEventProcessor.java:168)
[23:54:54 ERROR]: com.lmax.disruptor.BatchEventProcessor.run(BatchEventProcessor.java:125)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[23:54:54 ERROR]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[23:54:54 ERROR]: java.lang.Thread.run(Thread.java:748)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Signal Dispatcher
[23:54:54 ERROR]: PID: 4 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Finalizer
[23:54:54 ERROR]: PID: 3 | Suspended: false | Native: false | State: RUNNABLE
[23:54:54 ERROR]: Thread is waiting on monitor(s):
[23:54:54 ERROR]: Locked on:java.util.zip.Inflater.end(Inflater.java:373)
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.util.zip.Inflater.end(Native Method)
[23:54:54 ERROR]: java.util.zip.Inflater.end(Inflater.java:373)
[23:54:54 ERROR]: java.util.zip.Inflater.finalize(Inflater.java:383)
[23:54:54 ERROR]: java.lang.System$2.invokeFinalize(System.java:1270)
[23:54:54 ERROR]: java.lang.ref.Finalizer.runFinalizer(Finalizer.java:102)
[23:54:54 ERROR]: java.lang.ref.Finalizer.access$100(Finalizer.java:34)
[23:54:54 ERROR]: java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:217)
[23:54:54 ERROR]: ------------------------------
[23:54:54 ERROR]: Current Thread: Reference Handler
[23:54:54 ERROR]: PID: 2 | Suspended: false | Native: false | State: WAITING
[23:54:54 ERROR]: Stack:
[23:54:54 ERROR]: java.lang.Object.wait(Native Method)
[23:54:54 ERROR]: java.lang.Object.wait(Object.java:502)
[23:54:54 ERROR]: java.lang.ref.Reference.tryHandlePending(Reference.java:191)
[23:54:54 ERROR]: java.lang.ref.Reference$ReferenceHandler.run(Reference.java:153)
[23:54:54 ERROR]: ------------------------------
```

### Plugin list:
- Dynmap
- WorldBorder

### Actions to reproduce (if known):
None

### Paper build number:
624

### Context:
One player online, digging dirt for a while, suddenly timed out and a few minutes later that appeared in console.

**Comment by jason-s-yu** [comment]
I'm also getting this error repeatedly on 1.14.2-65

**Comment by leifadev** [comment]
Same here, when I reload mt server this happens

**Comment by aikar** [comment]
Closing due to 1.14, please reopen a new ticket if still persists on 1.15.2 latest
